br/backup-and-restore-storages.md

```diff
@@ -46,37 +67,81 @@ Cloud storages such as S3 and GCS sometimes require additional configuration for
 
 > **Note:**
 >
-> It is not recommended to pass in the access key and secret access key directly in the storage URL, because these keys are logged in plain text. BR tries to infer these keys from the environment in the following order:
+> It is not recommended to pass in the access key and secret access key directly in the storage URL, because these keys are logged in plain text. The migration tool tries to infer these keys from the environment in the following order:
```

Comment:
```suggestion
> It is not recommended to pass in the access key and secret access key directly in the storage URL, because these keys are logged in plain text. Each migration tool tries to infer these keys from the environment in the following order:
```